pkg/networkoptions/network_options.go

```diff
@@ -126,6 +128,35 @@ func AddNetworkFlagsToCmd(cmd *cobra.Command, networkFlags *NetworkFlags, addEnd
 	}
 }
 
+func GetNetworkFlagsGroup(cmd *cobra.Command, networkFlags *NetworkFlags, addEndpoint bool, supportedNetworkOptions []NetworkOption) flags.GroupedFlags {
```

Comment:
This should replace the existing  AddNetworkFlagsToCmd command that is used by other commands right now